README.md

```diff
@@ -101,6 +101,8 @@ In `config/config.exs`
 
 ```elixir
 if Mix.env() != :prod do
+  use Mix.Config
+
```

Comment:
Thanks for the PR! Could we extract it outside the condition? It will be more clear since this is expected to be present in any config file.

```suggestion
use Mix.Config

# somewhere in your config file
if Mix.env() != :prod do
```

Reply:
A lot of tooling inside github, not sure if I selected the correct option in how to include your suggestion. =)